op/framework.py

```diff
@@ -156,9 +156,7 @@ def emit(self, *args, **kwargs):
         The current storage state is committed before and after each observer is notified.
         """
         framework = self.emitter.framework
-        # TODO This needs to be persisted.
-        framework._event_count += 1
-        key = str(framework._event_count)
+        key = str(framework._stored['event_count'])
```

Comment:
Sorry for not noticing this earlier, but this doesn't feel great. It's picking up the count and assuming a different location is incrementing it. Splits the knowledge about something that is very tightly coupled for no apparent reason. The other line is completely alone inside _emit. Can we please merge both actions here again?